docs/source/install/vagrant.rst

```diff
@@ -0,0 +1,117 @@
+Vagrant / OVA is a quick and easy way to try |st2| in a VM.
+
+It's already pre-installed, configured and tested virtual appliance image and so saves time avoiding
+time-consuming installation and configuration steps. Perfect for a quick platform overview,
+testing, demo or even using |st2| in isolated from the internet air-gapped systems.
+
+We highly recommend using a Vagrant box to get familiar with the |st2| platform.
+
+.. contents:: Contents
+   :local:
+
+The source code is available as a GitHub repo:
+`StackStorm/packer-st2 <https://github.com/StackStorm/packer-st2>`_
+
+---------------------------
+
+Requirements
+------------
+`Vagrant <https://www.vagrantup.com/>`_ and `Virtualbox <https://www.virtualbox.org/>`_ are required.
+If you're not familiar with Vagrant, we recommend looking at `Introduction to Vagrant <https://www.vagrantup.com/intro/index.html>`_.
+
+Quick Start
+-----------
+Starting a |st2| Vagrant VM is easy:
+
+.. code-block:: bash
+
+  vagrant init stackstorm/st2
+  vagrant up
+  vagrant ssh
+
+
+Virtual Appliance / OVA
+-----------------------
+As alternative to Vagrant box is Virtual appliance which is available for download as ``.OVA``
+image from the `StackStorm/packer-st2 Github Releases <https://github.com/StackStorm/packer-st2/releases>`_
+page. It might be especially helpful for the isolated from the internet air-gapped environments.
+
+.. note::
+
+  | *Linux login credentials:*
+  | Username: ``vagrant``
+  | Password: ``vagrant``
+  |
+  | *StackStorm login details:*
+  | Username: ``st2admin``
+  | Password: ``Ch@ngeMe``
+
+.. warning::
+
+    If using OVA in production environment, don't forget to change the default credentials
+    and delete SSH authorized keys for ``vagrant`` linux user.
+
+
+At the moment only Virtualbox provider is supported. VMWare-compatible virtual appliance is
+available with `StackStorm Enterprise (EWC) <https://stackstorm.com/#product>`_ image.
+Ask `StackStorm Support <support@stackstorm.com>`_ for more info.
+
+
+Tips & Tricks
+-------------
+Updating the Vagrant box
+~~~~~~~~~~~~~~~~~~~~~~~~
+Once the newer box version is released, Vagrant will warn you about the available update.
+To update the box:
+
+.. code-block:: bash
+
+    vagrant box outdated
+    vagrant box remove stackstorm/st2
+    vagrant up
+
+
+Pinning the Vagrant box version
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Whether you want to pin |st2|, it's possible to use specific box version.
+While adding the box for the first time:
+
+.. code-block:: bash
+
+    vagrant init stackstorm/st2 --box-version 2.7.1-20180507
+    vagrant up
+
+Or directly in ``Vagrantfile``:
+
+.. code-block:: ruby
+
+    Vagrant.configure("2") do |config|
+      config.vm.box = "stackstorm/st2"
+      config.vm.box_version = "2.7.1-20180507"
+    end
+
+The list of available box versions can be found at `Vagrant Cloud <https://app.vagrantup.com/stackstorm/boxes/st2>`_.
+
+
+.. TODO: Add a Vagrant port forwarding example
```

Comment:
As figured out, we can automatically enable port forwarding rules for a shipped Vagrant box.
I think we should do it by default, make Web UI accessible for example via `localhost:9000` (considering st2 ports are 9001+).
This will simplify user experience a lot.

Created https://github.com/StackStorm/packer-st2/issues/28 to track that.

Reply:
Docs should not be merged until this is figured out. Second thing users will want to do is use Web UI.

Reply:
Alright, solved by https://github.com/StackStorm/packer-st2/pull/29.
Here is how it'll work and look like: 

![vagrant-up](https://user-images.githubusercontent.com/1533818/39960490-e51f14ae-561b-11e8-9df2-9f3983c92dfd.png)